Description: Extract the version from the Debian changelog
 Upstream uses the VCS to get the correct version in the build system, but we're using tarballs.
 This patch uses a similar mechanism but extracts the version from the packaging instead.
Author: Simon Chopin <simon.chopin@canonical.com>
Forwarded: not-needed
Last-Update: 2022-03-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: tpm2-tss/bootstrap
===================================================================
--- tpm2-tss.orig/bootstrap
+++ tpm2-tss/bootstrap
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-git describe --tags --always --dirty > VERSION
+dpkg-parsechangelog -S Version | sed 's/\(.*\)-[^-]*/\1/' > VERSION
 
 # generate list of source files for use in Makefile.am
 # if you add new source files, you must run ./bootstrap again
